src/gate_detect/README.md

```diff
@@ -0,0 +1,88 @@
+##Purpose
+
+This node takes in raw image data from a camera and determines whether there is a gate in the image, and if so what the 
+orientation of the gate is.
```

Comment:
Technicality, but this is a package, not just a node. If all this is referring to a node, should probably be under it's own heading.

Reply:
fixed